src/js/components/PanelBody/PanelBody.js

```diff
@@ -0,0 +1,31 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class PanelBody extends PureComponent {
+  render () {
+    const { className, mods, style, children } = this.props
+
+    return (
+      <div className={ getClassName(className, mods) } style={ style }>
+        { children }
+      </div>
+    )
+  }
+}
+
+
```

Comment:
style nit: There are some places with 2 newlines and some without? Any particular reason?

Reply:
Adding a specified linter with the spacing would fix your concerns here.  i don't think it's a big deal.  🤷‍♂️   But we can agree on a linter say standard, airbnb or another.